services/containers/service.go

```diff
@@ -13,10 +13,18 @@ import (
 )
 
 func init() {
-	plugin.Register("containers-grpc", &plugin.Registration{
+	plugin.Register(&plugin.Registration{
 		Type: plugin.GRPCPlugin,
+		ID:   "io.containerd.containers-grpc",
```

Comment:
I just used what we registered as before right now.   What do you think we should do for naming the rest of these?  I don't like it as is right now.